source/analysis/REIXS/REIXSXESImageInterpolationAB.cpp

```diff
@@ -303,6 +303,7 @@ void REIXSXESImageInterpolationAB::setInputDataSourcesImplementation(const QList
 	cachedValues_ = QVector<double>(axes_.at(0).size);
 	axisValueCacheInvalid_ = true;
 	cachedAxisValues_ = QVector<double>(axes_.at(0).size);
+	onInputSourceSizeChanged();

```

Comment:
@dretrex ,
The connection on line 297 seems to be bad.  I have found in the past that the argument mis-match can cause a silent connection failure.  However, when I tried to make ```onInputSourceSizeChanged()``` into ```onInputSourceSizeChanged(int)``` so that they do match, it still doesn't work.  ```emitSizeChanged(0)``` is calling the ```AMDatasource::emitSizeChanged(int)```, which is calling ```AMDataSignalSource::emitSizeChanged(int)```, but the chain stops there.  It never reaches ```onInputSourceSizeChanged()``` or ```onInputSourceSizeChanged(int)```   (and ```onInputSourceSizeChanged()``` never paid attention to the argument, and only looks at axis 0.